### PR TITLE
[DO NOT REVIEW][DO NOT SUBMIT] Experimental avoiding serialization

### PR DESCRIPTION
**Pull request by oxddr**
Empty for now.

This PR is based on work of @wojtek-t in #60067, but rebased onto newer state of master.

### CODE REVIEW

File: pkg/registry/core/node/storage/storage.go [hunk 1]
```diff
@@ -88,6 +89,13 @@ func NewStorage(optsGetter generic.RESTOptionsGetter, kubeletClientConfig client
 		ExportStrategy: node.Strategy,
 
 		TableConvertor: printerstorage.TableConvertor{TablePrinter: printers.NewTablePrinter().With(printersinternal.AddHandlers)},
+
+		SerializationSchemesToCache: []*runtime.SerializationScheme{
```
Comment (justinsb) [inline pkg/registry/core/node/storage/storage.go]: I like the explicitness, but we should also consider an LRU cache

Comment (justinsb) [reply pkg/registry/core/node/storage/storage.go]: (once we've verified this is a win)

**Comment by oxddr** [comment]
@mm4tt I've rebased this PR to the HEAD as of yesterday's afternoon.

**Pull request closed without merging**